votes/0147-3pointer-and-glorv-as-tikv-reviewers.md

```diff
@@ -0,0 +1,28 @@
+# A Vote for @3pointer and @glorv as TiKV Reviewers
+
+## Proposal
+
+@3pointer contributes to the Backup and Restore project, creates and helps on reviewing related pull requests on TiKV.
+
+* [Reviewed pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+reviewed-by%3A3pointer)
+* [Authored pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+author%3A3pointer)
+
+@glorv contributes to the TiDB-Lightning project, creates and helps on reviewing related pull requests on TiKV.
+
+* [Reviewed pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+reviewed-by%3Aglorv)
+* [Authored pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+author%3Aglorv)
+
+I (@overvenus) hereby nominate @3pointer and @glorv as TiKV Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 72 hours unless there is an objection or not enough votes.
+
+## Result
+
+Approved by X binding votes, X non-binding votes.
```

Comment:
```suggestion
Approved by 2 binding votes, 2 non-binding votes.
```

72 hours exceeded. @overvenus you may ask a maintainer to conclude the result and accept the vote by merging it.

Reply:
/unhold